internal/common/common_test.go

```diff
@@ -231,6 +231,25 @@ var _ = Describe("get hosts by role", func() {
 	})
 })
 
+var _ = Describe("compare OCP 4.10 versions", func() {
+	It("GA release", func() {
+		is410Version, _ := VersionGreaterOrEqual("4.10.0", "4.10.0-0.alpha")
+		Expect(is410Version).Should(BeTrue())
+	})
+	It("pre-release", func() {
+		is410Version, _ := VersionGreaterOrEqual("4.10.0-fc.1", "4.10.0-0.alpha")
+		Expect(is410Version).Should(BeTrue())
+	})
+	It("pre-release z-stream", func() {
+		is410Version, _ := VersionGreaterOrEqual("4.10.1-fc.1", "4.10.0-0.alpha")
+		Expect(is410Version).Should(BeTrue())
+	})
+	It("nightly release", func() {
+		is410Version, _ := VersionGreaterOrEqual("4.10.0-0.nightly-2022-01-23-013716", "4.10-alpha")
```

Comment:
```suggestion
		is410Version, _ := VersionGreaterOrEqual("4.10.0-0.nightly-2022-01-23-013716", "4.10.0-0.alpha")
```

Reply:
Yes, it's not just to check the unit-tests (something fails in my local env)

Reply:
looks like these tests are not run, seeing as the job passed

Reply:
And done now